### ISSUE TITLE
HistoryGraph Coloring Scheme

### ISSUE THREAD

**Issue by ScumbagSteve** [Zefau/ioBroker.jarvis#1677]
The coloring scheme of the HistoryGraph seems off.
The colors used on the x and y axis are found nowhere else and apparently do not reflect the site-wide color scheme set up in "theme settings". It looks like a light blue to me. I propose to make the color adjustable, or at least to use colors used in the theme.

![Unbenannt](https://user-images.githubusercontent.com/8447146/166079492-69bdb288-0fa5-4b96-9c10-2d0ff6cf2677.PNG)

**Comment by mcuiobroker** [comment]
Do you mean this Color?
![image](https://user-images.githubusercontent.com/84818934/166106923-14a00318-a767-461c-b636-7956a4b99f59.png)
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/module/historygraph/formatparameter#beispiel-text-farbe-von-achse-y-aendern

**Comment by ScumbagSteve** [comment]
Yes, this color. I am a pro user and can fix it with the parameters, however I think it should be fixed in general for anyone.
